gms/versioned_value.hh

```diff
@@ -10,15 +10,15 @@
 
 #pragma once
 
-#include <seastar/core/sstring.hh>
-#include "utils/serialization.hh"
-#include "locator/host_id.hh"
-#include "version_generator.hh"
-#include "gms/inet_address.hh"
+#include "cdc/generation_id.hh"
 #include "dht/token.hh"
+#include "gms/inet_address.hh"
+#include "locator/host_id.hh"
 #include "schema/schema_fwd.hh"
+#include "service/state_id.hh"
 #include "version.hh"
-#include "cdc/generation_id.hh"
+#include "version_generator.hh"
+#include <seastar/core/sstring.hh>
```

Comment:
some reorderings here, probably belong to the "cleanup" commit (other files as well)

Reply:
yes, moved. This is also coming from the .clang-format rules (that contain the header ordering rules, so when I added one the whole block has been reordered) - cc @kreuzerkrieg